limit_to_employees/README.rst

```diff
@@ -0,0 +1,13 @@
+===========================
+Limit To Employees
+===========================
+
+When user do following action in odoo, it only shows internal users contacts in dropdown.
+  * Chatter mentioning (@username)
+  * Helpdesk ticket assignments
+  * Invoicing, Salesperson
```

Comment:
Weren't there originally listed more? What happened to them?

Reply:
I not understand what changes we need to do in description?

Reply:
I not understand what changes we need to do in description?

Reply:
I think it must have been a mistake on my end.